src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
```

Comment:
it's a good practice to use object destructing. Either in parameter or in component body